MusicLister/MusicLister/ViewController.swift

```diff
@@ -8,13 +8,38 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UITableViewController {
 
+    let myArray = [Song(name: "Sea Shanty 2", artist: "Runescape", album: "Port Sarim", genre: "Kandarin", length: "3:50"),
```

Comment:
This variable could have a better name (maybe "songs") so it's easier to read below.